src/main/java/frc/trigon/robot/utilities/Conversions.java

```diff
@@ -0,0 +1,20 @@
+package frc.trigon.robot.utilities;
+
+public class Conversions {
+
+        /**
+        * Converts voltage to compensated power
+        *
+        * The voltage compensation saturation will determine what voltage represents 100% output.
+        *
+        * The compensated power is the power resulting from turning the voltage off and on without stopping.
+        * It's basically the voltage divided by the saturation of the compensation.
+        *
+        * @param voltage the voltage
```

Comment:
nope. the explanation for voltage is not "the voltage" please write proper documentation

Reply:
ANY PROBLEM????!!!!!!!!!????????!!!!!!!!!!!???????????!!!!!!!!!!!!!!!!?????????!!!!!!!!!!!!???????????!!!!!!!!!!!!!

Reply:
yes. you aren't explaining anything.

Reply:
@lycaon7 so do you mean that I have to write an entire paragraph about the voltage?

Reply:
not a paragraph. one sentence is enough.
your current documentation is "voltage is the voltage" do you not see the problem with that?

Reply:
What else if not the voltage?

Reply:
**explain what the parameter is.**

Reply:
Done